### PR TITLE
Add field name to users

### PR DESCRIPTION
**Pull request by ritamma**

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,2 +1,13 @@
...
+  protect_from_forgery with: :exception
+
+  before_action :configure_permitted_parameters, if: :devise_controller?
+
+  protected
+
+  def configure_permitted_parameters
+    devise_parameter_sanitizer.permit(:sign_up) { |u| u.permit(:name, :email, :password)}
+
+    devise_parameter_sanitizer.permit(:account_update) { |u| u.permit(:name, :email, :password, :current_password)}
```
Comment (fooooxmr) [inline app/controllers/application_controller.rb:11]: Check this line.

File: app/views/devise/registrations/edit.html.erb [hunk 2]
```diff
@@ -12,6 +12,11 @@
     <div>Currently waiting confirmation for: <%= resource.unconfirmed_email %></div>
   <% end %>
 
+  <div>
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autofocus: true %>
```
Comment (fooooxmr) [inline app/views/devise/registrations/edit.html.erb]: Check what do autofocus.

**Review by fooooxmr — CHANGES_REQUESTED** [review]

File: app/views/devise/registrations/edit.html.erb (hunk 2 above)
Comment (ritamma) [reply app/views/devise/registrations/edit.html.erb]: Done

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (ritamma) [reply app/controllers/application_controller.rb:11]: We can use account_update, if we want to update users data, we have default routes for this action.

**Review by fooooxmr — APPROVED** [review]

**Pull request merged**